milestone1/mvnw.cmd

```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```

Comment:
Same as mvnw. Do we need this?

Reply:
We can delete it no problem. Spring created it when initializing the project